app/models/edition/national_applicability.rb

```diff
@@ -12,10 +12,16 @@ def process_associations_before_save(edition)
   included do
     has_many :nation_inapplicabilities, foreign_key: :edition_id, dependent: :destroy, autosave: true
     validates_associated :nation_inapplicabilities
+    alias_method :excluded_nations, :nation_inapplicabilities
+    validates :excluded_nations, length: { maximum: Nation.all.count - 1, message: 'can not exclude all nations' }
 
     add_trait Trait
   end
 
+  def has_nation_applicabilities
```

Comment:
it sounds like a question `has_nation_applicabilities?`
Also this method seems to not be in use.